### ISSUE TITLE
Max size is too small if the clock is stretched (resized bigger)

### ISSUE THREAD

**Issue by inson1** [artyommironov/nanji#10]

**Comment by artyommironov** [comment]
Duplicates #18 ?

**Comment by inson1** [comment]
@artyommironov No, this issue is if the widget is stretched. #18 is for normal size

**Comment by inson1** [comment]
@artyommironov 
1) this option is more hot fix then full fix imho 
2) this optionis full fix, but yea I also hate this kind of work

I think hot fix is better than no fix, but I would let one issue open for that and maybe one day somebody will fix it
